externs/firebaseui-externs.js

```diff
@@ -326,20 +330,30 @@ firebaseui.auth.SignInOption = function() {};
  */
 firebaseui.auth.SignInOption.prototype.provider;
 
+/**
+ * Defines the sign-in option
+ * needed to configure the FirebaseUI federated sign-in widget.
```

Comment:
Limit is 80 characters per line.
Try to fit as much as possible per line.

Reply:
Isn't this ok about line length limit?

Reply:
It's better to fit into the first line as much as possible:
 * Defines the sign-in option needed to configure the FirebaseUI federated
 * sign-in widget.

Reply:
Oh, I got it.

Reply:
It doesn't look like you changed it to @wti806's recommendation.